Modules/seek/Controller/SeekbookingController.php

```diff
@@ -471,37 +465,30 @@ private function isNew($param) {
 
     public function deleteAction($id_space, $id){
         
-        $sendmail = $this->request->getParameter("sendmail");
+        $sendEmail = $this->request->getParameter("sendmail");
         
-        if( $sendmail == 1){
+        if( $sendEmail == 1){
```

Comment:
same comment as above for comparison , sendmail = "1" no ? should work, but bad practive to compare string to integer